packages/google-pay-integration/src/gateways/google-pay-checkoutcom-gateway.ts

```diff
@@ -0,0 +1,100 @@
+import { RequestSender } from '@bigcommerce/request-sender';
+
+import {
+    ContentType,
+    InvalidArgumentError,
+    isRequestError,
+    PaymentIntegrationService,
+} from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import assertIsGooglePayCheckoutComInitializationData from '../guards/is-google-pay-checkoutcom-initialization-data';
+import { isGooglePayThreeDSecureResult } from '../guards/is-google-pay-threedsecure-result';
+import assertIsGooglePayTokenObject from '../guards/is-google-pay-token-object';
+import {
+    GooglePayAdditionalActionProcessable,
+    GooglePayCardDataResponse,
+    GooglePayCheckoutComTokenObject,
+    GooglePayGatewayParameters,
+    GooglePaySetExternalCheckoutData,
+    GooglePayTokenObject,
+} from '../types';
+
+import GooglePayGateway from './google-pay-gateway';
+
+export default class GooglePayCheckoutComGateway
+    extends GooglePayGateway
+    implements GooglePayAdditionalActionProcessable
+{
+    constructor(service: PaymentIntegrationService, private _requestSender: RequestSender) {
+        super('checkoutltd', service);
```

Comment:
Howcome this is not called checkoutcom?

Reply:
No idea, the gateway id is determined by Google and Checkout[dot]Com, it's not a name chosen by me. 🤷🏻‍♂️